### PR TITLE
Translate Assign Pod Node to Bahasa Indonesia

### PR DESCRIPTION
**Pull request by marchellametta**
Addresses #15341
@irvifa @girikuncoro

### CODE REVIEW

**Comment by marchellametta** [comment]
Signed a Contributor License Agreement (CLA) as requirement

File: content/id/docs/concepts/configuration/assign-pod-node.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+Semua `matchExpressions` berkaitan dengan affinity dan anti-affinity `requiredDuringSchedulingIgnoredDuringExecution` harus dipenuhi agar pod dapat dijadwalkan pada node.
+
+#### More Practical Use-cases
+
+Affinity and anti-affinity antar pod dapat menjadi lebih berguna saat digunakan bersamaan dengan koleksi dengan level yang lebih tinggi seperti ReplicaSets, StatefulSets, Deployments, dll. Pengguna dapat dengan mudah mengkonfigurasi bahwa satu set workload harus
+ditempatkan bersama dalam topologi yang didefinisikan sama, misalnya, node yang sama. 
+
+##### Selalu ditempatkan bersamaan pada node yang sama
+
+Dalam kluster berisi 3 node, sebuah aplikasi web memiliki in-memory cache seperti redis. Kita mau web-server dari aplikasi ini sebisa mungkin ditempatkan bersamaan dengan cache.
```
Comment (irvifa) [inline content/id/docs/concepts/configuration/assign-pod-node.md]:
```suggestion
Dalam kluster berisi 3 node, sebuah aplikasi web memiliki in-memory cache seperti redis. Kita menginginkan agar  _web-server_ dari aplikasi ini sebisa mungkin ditempatkan bersamaan dengan cache.
```

File: content/id/docs/concepts/configuration/assign-pod-node.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+
+Berikut ini potongan yaml dari deployment redis sederhana dengan 3 replika dan label selector `app=store`, Deployment memiliki konfigurasi `PodAntiAffinity` untuk memastikan scheduler tidak menempatkan replika bersamaan pada satu node.
```
Comment (irvifa) [inline content/id/docs/concepts/configuration/assign-pod-node.md]:
```suggestion
Berikut ini kutipan yaml dari deployment redis sederhana dengan 3 replika dan label selector `app=store`, Deployment memiliki konfigurasi `PodAntiAffinity` untuk memastikan _scheduler_ tidak menempatkan replika bersamaan pada satu node.
```

File: content/id/docs/concepts/configuration/assign-pod-node.md [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+                operator: In
+                values:
+                - store
+            topologyKey: "kubernetes.io/hostname"
+      containers:
+      - name: redis-server
+        image: redis:3.2-alpine
+```
+
+Potongan yaml dari deployment webserver berikut ini memiliki konfigurasi `podAntiAffinity` dan `podAffinity`. Konfigurasi ini menginformasikan scheduler bahwa semua replika harus ditempatkan bersamaan dengan pod yang memiliki label selector `app=store`. Konfigurasi ini juga memastikan bahwa setiap replika webserver tidak ditempatkan bersamaan pada satu node.
```
Comment (irvifa) [inline content/id/docs/concepts/configuration/assign-pod-node.md]:
```suggestion
Kutipan yaml dari deployment webserver berikut ini memiliki konfigurasi `podAntiAffinity` dan `podAffinity`. Konfigurasi ini menginformasikan scheduler bahwa semua replika harus ditempatkan bersamaan dengan pod yang memiliki label selector `app=store`. Konfigurasi ini juga memastikan bahwa setiap replika webserver tidak ditempatkan bersamaan pada satu node.
```

File: content/id/docs/concepts/configuration/assign-pod-node.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+
+##### Tidak akan pernah ditempatkan bersamaan dalam node yang sama
+
+
+Contoh di atas menggunakan aturan `PodAntiAffinity` dengan` topologyKey: "kubernetes.io/hostname"` untuk melakukan deploy kluster redis sehingga tidak ada dua instance terletak pada hos yang sama.
+Lihat [tutorial ZooKeeper](/docs/tutorials/stateful-application/zookeeper/#tolerating-node-failure)  untuk contoh dari konfigurasi StatefulSet dengan anti-affinity untuk ketersediaan tinggi, menggunakan teknik yang sama. 
+
+Untuk informasi lebih lanjut tentang affinity/anti-affinity antar pod, lihat [design doc](https://git.k8s.io/community/contributors/design-proposals/scheduling/podaffinity.md). 
+
+Anda juga dapat mengecek [Taints](/docs/concepts/configuration/taint-and-toleration/), yang memungkinkan sebuah *node* untuk *menolak* sekumpulan pod.
```
Comment (irvifa) [inline content/id/docs/concepts/configuration/assign-pod-node.md]:
```suggestion
Kamu juga dapat mengecek [Taints](/docs/concepts/configuration/taint-and-toleration/), yang memungkinkan sebuah *node* untuk *menolak* sekumpulan pod.
```

File: content/id/docs/concepts/configuration/assign-pod-node.md [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+
+## nodeName
+
+`nodeName` adalah bentuk paling sederhana dari pembatasan pemilihan node, tetapi karena
+keterbatasannya biasanya tidak digunakan. `nodeName` adalah sebuah field dari
```
Comment (irvifa) [inline content/id/docs/concepts/configuration/assign-pod-node.md]:
```suggestion
keterbatasannya biasanya tidak digunakan. `nodeName` adalah sebuah _field_ dari
```

File: content/id/docs/concepts/configuration/assign-pod-node.md [hunk 6]
```diff
@@ -0,0 +1,324 @@
...
+PodSpec.  Jika tidak kosong, scheduler mengabaikan pod dan
```
Comment (irvifa) [inline content/id/docs/concepts/configuration/assign-pod-node.md]:
```suggestion
PodSpec. Jika tidak kosong, scheduler mengabaikan pod dan
```

**Comment by irvifa** [comment]
Thanks @marchellametta 🎉 
/lgtm

**Pull request merged**